webextension/background.js

```diff
@@ -72,3 +73,28 @@ function disableAddon(tabId) {
   browser.browserAction.disable(tabId);
   browser.browserAction.setTitle({ tabId, title: "Containers disabled in Private Browsing Mode" });
 }
+
+browser.tabs.onCreated.addListener(tab => {
```

Comment:
Either that or for efficiency maybe we have an that does all the browser.apiName obj call methods to other objects for both the assignment and counting... however I dunno if we have a perf issue.

Like:
```
const apiObj = {
  init() {
    browser.webRequest.onCompleted.addListener((details) => {
      tabPageCounters.request.onComplete(....)
      assignManager.calculateContextMenu(...)
    });
  }
}
```